dist/Storable/Storable.xs

```diff
@@ -296,6 +296,28 @@ typedef STRLEN ntag_t;
 #define VSTRING_CROAK() CROAK(("Cannot retrieve vstring in this perl"))
 #endif
 
+#ifndef sv_vstring_get
+#define sv_vstring_get(sv,pvp,lenp) S_sv_vstring_get(aTHX_ sv,pvp,lenp)
+static bool S_sv_vstring_get(pTHX_ SV *sv, const char **pvp, STRLEN *lenp)
+{
+  MAGIC *mg;
+  if(!SvMAGICAL(sv) || !(mg = mg_find(sv, PERL_MAGIC_vstring)))
+    return FALSE;
+
+  *pvp  = mg->mg_ptr;
+  *lenp = mg->mg_len;
+  return TRUE;
+}
+#endif
+
+#ifndef sv_vstring_set
+#define sv_vstring_set(sv,pv,len) \
+  STMT_START {                                             \
+    sv_magic((sv), NULL, PERL_MAGIC_vstring, (pv), (len)); \
+    SvRMAGICAL_on(sv);                                     \
+  } STMT_END
+#endif
+
```

Comment:
This belongs in Devel::PPPort

Reply:
That sounds like an excellent idea that I fully support someone else doing.

Really - I have no idea how to edit or maintain PPPort. The entire process seems utterly opaque to me. If someone else wants to copypaste it over there then I can update this PR to match.

Reply:
There is a more general problem that almost no-one knows how to do that. I suspect that the real solution is that several of us learn to do so because otherwise this keeps being a recurring problem.

Reply:
I can help getting this into D:P

Reply:
It sounds like overall the process for adding this to `D::P` is nontrivial enough, that I'd suggest it might be easier to first merge this PR as-is, and then as a separate change, move these support helpers out to there. That way it doesn't hold up this particular change and the work depending on it.